package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simorgh",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Apologies for the confusion regarding changelogs and version number updates. It's now been clarified that we won't have a Changelog file and won't increment the Simorgh package version in PRs.